lib/bullhorn/broadway.ex

```diff
@@ -56,21 +58,28 @@ defmodule Bullhorn.Broadway do
     [failed_message]
   end
 
-  defp notify_handler({type, message}) do
-    {handler, _messages} =
-      Enum.find(message_handlers(), fn {_handler, message_types} ->
-        type in message_types
-      end)
+  defp notify_handler({:user_created, message}) do
+    Logger.debug("Handling User Created message")
+    Users.created(message)
+  end
 
-    case handler do
-      nil ->
-        Logger.debug("Ignored #{type} message")
+  defp notify_handler({:password_changed, message}) do
+    Logger.debug("Handling Password Changed message")
+    Users.password_changed(message)
+  end
 
-      mod ->
-        Logger.debug("Handling #{type} message")
-        apply(mod, :handle_message, [message])
-    end
+  defp notify_handler({:password_reset, message}) do
+    Logger.debug("Handling Password Reset message")
+    Users.password_reset(message)
   end
 
-  defp message_handlers, do: Application.get_env(:bullhorn, :message_handlers)
+  defp notify_handler({:tribble_failed, message}) do
+    Logger.debug("Handling Tribble Failed message")
+    Orders.tribble_failed(message)
+  end
+
+  defp notify_handler(_) do
+    Logger.debug("Ignoring message")
```

Comment:
Just to help debugging if need be

```suggestion
  defp notify_handler({event, _message}) do
    Logger.debug("Ignoring #{event} message")
```